transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/widget/PhoneWidget.java

```diff
@@ -76,6 +76,7 @@ public void onFocusChange(View v, boolean hasFocus) {
             editText.setText(mDefaultValue);
             editText.setInputType(InputType.TYPE_CLASS_PHONE);
             editText.setOnKeyListener(new DefaultKeyListener(mDefaultFocusView, editText));
+            editText.addTextChangedListener(new TextChangeListener());
```

Comment:
PhoneWidget already has a `TextChangedListener`

Reply:
https://developer.android.com/reference/android/view/View.OnKeyListener
OnKeyListener not triggered in some keyboard, google recommended to use textchange listener for text changes.

Reply:
fixed